guidelines/enhancement_template.md

```diff
@@ -72,17 +72,33 @@ this proposal. Describe why the change is important and the benefits to users.
 
 ### User Stories
 
-Detail the things that people will be able to do if this is
-implemented. In each story, explain who the actor is based on their
-role, explain what they want to do with the system, and explain the
-underlying goal they have. Use the standard three part formula:
+Detail the things that people will be able to do if this is implemented and
+what goal that allows them to achieve .In each story, explain who the actor
+is based on their role, explain what they want to do with the system,
+and explain the underlying goal they have, what it is they are going to
+achieve with this new feature.
 
-"As a <role>, I want to <take some action> so that I can <accomplish a
-goal>."
+Use the standard three part formula:
+
+"As a \<role\>, I want to \<take some action\> so that I can \<accomplish a
+goal\>."
 
 Make the change feel real for users, without getting bogged down in
 implementation details.
 
+Here are some example user stories to show what they might look like:
+
+* As an OpenShift engineer, I want to write an enhancement, so that I can share my feature idea
+and get feedback from others.
+* As a product manager, I want the engineers to implement this RFE, so that my customer remains happy.
+* As an administrator, I want a one-click OpenShift installer, so that I can easily set up a new cluster
+without having to follow a long set of operations.
```

Comment:
A few more "real" examples like the administrator one might help folks, specific example:

"As an administrator installing OpenShift, I want the ability to specify static IP addresses for the hosts, so that I can operate a cluster in environments where DHCP is not available to automatically configure host networking"

Another option might be to take some "good" examples from existing enhancements and highlight them here as good practice?

Reply:
Retroactively rephrasing the motivation section for [conditional update risks][1] in this user-story pattern:

* As a cluster administrator, I would like to be able to update my cluster when it is safe to do so, but avoid updating my cluster into known regressions, because I like getting new features and bugfixes, but I do not like getting new regressions.
* As an OpenShift update administrator, I would like to declare conditional update risks, because sometimes regressions impact a subset of the fleet, and we want to encourage unaffected clusters to update (new features and bugfixes!) while encouraging vulnerable clusters to not update (avoiding regressions!).

[1]: https://github.com/openshift/enhancements/blob/master/enhancements/update/targeted-update-edge-blocking.md

Reply:
I like the idea of pulling a few good examples from existing enhancements. You could even link to the full enhancement docs.